### PR TITLE
feat(lock): add additional info

### PR DESCRIPTION
**Pull request by stonebuzz**
add additional info about ```locked fields``` and ```locked items```

image

image

| Q             | A
| ------------- | ---
| Bug fix?      | yes/no
| New feature?  | yes/no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #number

### CODE REVIEW

File: src/Lock.php [hunk 1]
```diff
@@ -252,6 +269,14 @@ public static function showForItem(CommonDBTM $item)
         ]);
         $header |= $results['header'];
 
+        echo "<div class='alert alert-primary d-flex align-items-center' role='alert'>";
+        echo "<i class='fas fa-info-circle fa-xl'></i>";
+        echo "<span class='ms-2'>";
+        echo __("Here the locked items are displayed.<br> A locked items is a manually deleted items (example: a monitor).
```
Comment (trasher) [inline src/Lock.php:275]:
```suggestion
        echo __("A locked item is a manually deleted item (example: a monitor).
```

File: src/Lock.php [hunk 2]
```diff
@@ -156,87 +145,115 @@ public static function showForItem(CommonDBTM $item)
...
-                echo "<td class='left'>" . $row['value'] . "</td>";
-                echo "</tr>\n";
+                echo "</table>";
+                $massiveactionparams['ontop'] = false;
+                Html::showMassiveActions($massiveactionparams);
+                Html::closeForm();
+            } else {
+                echo "<table class='tab_cadre_fixehov'>";
+                echo "<tbody>";
+                echo "<tr><th colspan='5' class='center'>" . __('Locked fields') . "</th></tr>";
```
Comment (trasher) [inline src/Lock.php:251]:
```suggestion
                echo "<tr><th colspan='5' class='center'>" . _n('Locked field', 'Locked fields') . "</th></tr>";
```

File: src/Lock.php [hunk 3]
```diff
@@ -73,27 +73,16 @@ public static function showForItem(CommonDBTM $item)
             return false;
         }
 
+        echo "<div class='alert alert-primary d-flex align-items-center' role='alert'>";
+        echo "<i class='fas fa-info-circle fa-xl'></i>";
+        echo "<span class='ms-2'>";
+        echo __("Here the locked fields are displayed.<br> A locked field is a manually modified field.
```
Comment (trasher) [inline src/Lock.php:79]:
```suggestion
        echo __("A locked field is a manually modified field.
```

File: src/Lock.php (hunk 1 above)
Comment (trasher) [reply src/Lock.php:275]: Maybe should you use anther concatenated string for second sentence instead of using an html tag in translations.

**Comment by stonebuzz** [comment]
image

File: src/Lock.php (hunk 1 above)
Comment (stonebuzz) [reply src/Lock.php]: fixed

File: src/Lock.php (hunk 2 above)
Comment (stonebuzz) [reply src/Lock.php]: fixed

File: src/Lock.php (hunk 3 above)
Comment (stonebuzz) [reply src/Lock.php]: fixed

File: src/Lock.php [hunk 4]
```diff
@@ -73,27 +73,17 @@ public static function showForItem(CommonDBTM $item)
             return false;
         }
 
+        echo "<div class='alert alert-primary d-flex align-items-center' role='alert'>";
```
Comment (trasher) [inline src/Lock.php:76]: Should not this messages be hidden if there are no locked fields (condition in iterator count)?

File: src/Lock.php [hunk 5]
```diff
@@ -252,6 +270,15 @@ public static function showForItem(CommonDBTM $item)
         ]);
         $header |= $results['header'];
 
+        echo "<div class='alert alert-primary d-flex align-items-center' role='alert'>";
```
Comment (trasher) [inline src/Lock.php:273]: Should not this message be hidden if itemtype is not computer?

**Review by trasher — COMMENTED** [review]
OK for me appart of that :)

File: src/Lock.php (hunk 5 above)
Comment (stonebuzz) [reply src/Lock.php:273]: nop, because ```Item_SoftwareVersion``` and ```Item_SoftwareLicense``` are displayed (if available) regardless of the itemtype

**Review by trasher — APPROVED** [review]

**Review by orthagh — COMMENTED** [review]
Seems good for me, but maybe add a little spacing between the 2nd message and the above table

**Pull request merged**